### DISCUSSION TITLE
Welcome to scblogs Discussions!

### DISCUSSION THREAD

**Discussion by stick-i** [stick-i/scblogs#63] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zdl000000** [comment]
大佬，您好！不好意思，我可以用这个博客项目做为毕设吗，不会用作其他商业用途，只是学习参考。请问这个博客项目要怎么开发？

**Reply by stick-i** [reply]
1. 你可以用这个项目的代码作为你的毕业设计，参考学习是没有问题的。
2. 如果你要对这个项目进行开发，你需要有相关的知识基础，比如SpringCloud和微服务相关的一些东西，还有中间件的使用方法。
    如果你已经掌握这些知识，你可以参考项目贡献者文档进行开发，https://github.com/stick-i/scblogs/blob/main/CONTRIBUTING.md
3. 另外，你还可以加入我们的项目群进行交流，群号码 729931395
